Ginger/GingerCoreNET/Run/RunsetExecutor.cs

```diff
@@ -641,12 +659,79 @@ public async Task RunRunset(bool doContinueRun = false)
                     await Runners[0].Executor.ExecutionLoggerManager.PublishToCentralDBAsync(RunSetConfig.LiteDbId, RunSetConfig.ExecutionID ?? Guid.Empty);
                 }
             }
+            catch(Exception ex)
+            {
+                Reporter.ToLog(eLogLevel.ERROR, "Exception occured when trying to Run runset ", ex);
+            }
             finally
             {
                 mRunSetConfig.IsRunning = false;
+            
             }
         }
 
+        private bool CheckforReRunConfig()
+        {
+            bool Result = true;
+            WorkSpace.Instance.RunsetExecutor.RunSetConfig.FailedBFGuidList = new List<Guid?>();
+            if (RunSetConfig.ReRunConfigurations.ReferenceExecutionID != null)
```

Comment:
Add validation to check if RunSetConfig.ReRunConfigurations.ReferenceExecutionID is valid GUID